heartbeat/mysql

```diff
@@ -544,14 +568,20 @@ set_master() {
     # Informs the MySQL server of the master to replicate
     # from. Accepts one mandatory argument which must contain the host
     # name of the new master host. The master must either be unchanged
-    # from the laste master the slave replicated from, or freshly
+    # from the last master the slave replicated from, or freshly
     # reset with RESET MASTER.
+    if [ -n "$OCF_RESKEY_replication_master_ssl_ca" -a -n "$OCF_RESKEY_replication_master_ssl_cert" -a -n "$OCF_RESKEY_replication_master_ssl_key" ]; then
```

Comment:
You should use `&&` instead of `-a`:
https://wiki.bash-hackers.org/commands/classictest#why_you_should_avoid_using_-a_and_-o

Reply:
Hi ,

> You should use `&&` instead of `-a`:
Damn right... However, I was following the `Bash` style of this file, where the `&&` was used just once in `IF` clause...  Everywhere else is the `-a` in use..

Should I check the whole file and make appropriate changes?  Although, this is a bit out of the scope of this ticket...

Looking for your feedback.

Reply:
No need to do that, unless you find it causing issues.

We just try to avoid introducing more of it, and fix it if it's causing issues.